test/integration/feedback_test.lua

```diff
@@ -64,4 +65,5 @@ function g.test_feedback()
     t.assert_equals(msg.server_id, helpers.uuid('a', 'a', 1))
     t.assert_equals(msg.cluster_id, helpers.uuid('a'))
     t.assert_type(msg.rocks.cartridge, 'string')
+    t.assert_type(msg.rocks.http, 'string')
```

Comment:
This test isn't correct. Cartridge dev layout doesn't conform to layout of regular project, but it still passes.

Reply:
What do you mean?

I needed to assure, that rocks from the manifest will appear in the feedback report. I can't check exact versions and underlaying manifest to a random location would be tedious. So I decided to check for the present of `http` module in report, since it is one of the most stable dependencies for cartridge and test are supposed to run under cartridge environment.